scene/gui/popup_menu.h

```diff
@@ -218,6 +221,9 @@ class PopupMenu : public Popup {
 
 	String bind_global_menu();
 	void unbind_global_menu();
+	bool is_special_menu() const;
+	void set_special_menu_root(const String &p_special);
+	String get_special_menu_root() const;
```

Comment:
I think calling it special is ambiguous, doesn't really carry enough information when you should use it. Isn't this always related to global menus? Should we not refer to them in some way with the name of these new methods?

Alternatively, how about calling it system menu? Though I'd rather we didn't add more new terms if we could.

Reply:
This refers to one of the aforementioned hardcoded menu names. A menu that is created/populated by OS and not fully customizable (it's possible to add items, but the menu itself can't be removed, moved, or set as submenu), in contrast to the user created one.

Reply:
One of the options (`_dock`) is a right click on the taskbar icon. Another is adding subitems of the `Apple` (and similarly `Help` and `Window`) menu (one mentioned in https://github.com/godotengine/godot/pull/83987#discussion_r1425362343), so it not always part of global menu. But indeed is always a system menu.

Reply:
Let's go with the `system` in the name then, if you don't mind.

Reply:
> Let's go with system in the name then, if you don't mind.

OK, will rename it a bit later.